tests/test_features.py

```diff
@@ -10,14 +15,6 @@
 except KeyError:
     pass
 
-import matplotlib
```

Comment:
Yeah, sorry for the confusion.  It's been a long time since I wrote that function, and my head has been in more standard numpy dimensional conventions more recently.



> I just tried the cens feature tests with the master-branch normalization and it passes so I would suggest we close this one and figure the normalization out in a separate PR?

Just to be clear, none of the previous normalization behavior has changed, right?

Otherwise, I think we're in good shape.  There are still some unresolved line comments (docstrings mainly) that I'd like to see resolved before merging.